### PR TITLE
Allow parsing `struct`s

### LINKED ISSUE #118: Structs cannot be included in DTOs
Trying to generate a contract containing a `struct` type produces error `[CNTR0004] Internal type `System.ValueType` is not known.`

### PR DESCRIPTION
**Pull request by mishioo**
I added `System.ValueType` to known types, Not sure if that's really all that should be done, but it seems to work correctly with `struct`s now.

Resolves  #118.

### CODE REVIEW

**Review by jakubfijalkowski — CHANGES_REQUESTED** [review]

File: contracts.proto [hunk 1]
```diff
@@ -10,6 +10,7 @@ enum KnownType {
   Guid = 3;
   Uri = 4;
   Boolean = 5;
+  ValueType = 6;
```
[the remark this answers is not in the document]
Comment (mishioo) [reply contracts.proto:13]: Ah, of course, I messed that up. :facepalm:

File: examples/properties/struct.cs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+public class DTO
```
Comment (jakubfijalkowski) [inline examples/properties/struct.cs:1]: This seems like it tests something that is already tested.

File: src/LeanCode.ContractsGenerator/Generation/TypeRefFactory.cs [hunk 3]
```diff
@@ -85,6 +85,7 @@ public TypeRef From(ITypeSymbol symbol)
             { ContainingNamespace.Name: "System", Name: "Guid" } => New(KnownType.Guid),
             { ContainingNamespace.Name: "System", Name: "Uri" } => New(KnownType.Uri),
             { ContainingNamespace.Name: "System", Name: "TimeSpan" } => New(KnownType.TimeSpan),
+            // { ContainingNamespace.Name: "System", Name: "ValueType" } => New(KnownType.ValueType),
```
Comment (jakubfijalkowski) [inline src/LeanCode.ContractsGenerator/Generation/TypeRefFactory.cs:88]: Should be removed.

**Review by jakubfijalkowski — CHANGES_REQUESTED** [review]

File: examples/properties/struct.cs (hunk 2 above)
Comment (mishioo) [reply examples/properties/struct.cs]: Should be `public struct DTO`, corrected.

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Allow running the generator in check-only mode,
 - _Invalid type_ analyzer,
 - Enum members (`EnumValue`) have attributes.
```
Comment (jakubfijalkowski) [inline CHANGELOG.md:19]: Let's change the last `.` to `,`.

**Review by jakubfijalkowski — APPROVED** [review]
Last nitpick, plus rebase, and we're good to go.

**Pull request merged**